src/components/dashboard.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import AppBar from 'material-ui/AppBar';
+import Register from './register';
+import style2 from './styling';
+import './styling.css';
+
+import FlatButton from 'material-ui/FlatButton';
+
+
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+import Reorder from 'material-ui/svg-icons/action/reorder';
+import LibraryBooks  from 'material-ui/svg-icons/av/library-books';
+
+import Divider from 'material-ui/Divider';
+import Reply from 'material-ui/svg-icons/content/reply';
+import FontIcon from 'material-ui/FontIcon';
+
+
+class Dashboard extends Component {
+    constructor(props){
+        super(props)
+    }
+    componentDidMount(){
+        if(!localStorage.getItem('token')){
+            this.props.history.push('/login')
+        }
+    }
+    handleLogOut =(e) =>{
+        localStorage.removeItem("token");
+        window.location.reload()
+        this.props.history.push('/login')
+
+    } 
+    render() {
+        return (
+        <div>
+            <div className = "center">
+                <h1 class = "text">Welcome To Yummy Recipes</h1> 
+            </div>
+        <div>
+        <Menu >
+            <MenuItem className = "menu_style" primaryText="Category" leftIcon={<Reorder />} />
+            <MenuItem className = "menu_style" primaryText="Recipe" leftIcon={<LibraryBooks />} />
+            <Divider  className = "menu_style" />
+            <MenuItem className = "menu_style" onClick = {this.handleLogOut}primaryText="Logout" leftIcon={<Reply />} />
+        </Menu>
+        </div>
+        </div>
+        )
+    }
+}
+ 
+export default Dashboard ;
+
+
+
+/*
```

Comment:
Avoid having commented out chunks of code

Reply:
Noted